package.json

```diff
@@ -1,22 +1,40 @@
 {
     "name": "openapi-codegen-typescript",
     "version": "0.1.0",
-    "description": "",
+    "description": "OpenApi codegen for generating types an mocks from swagger json file",
     "main": "dist/index.ts",
     "types": "dist/index.d.ts",
     "private": true,
+    "repository": "LandrAudio/openapi-codegen-typescript",
+    "author": {
```

Comment:
Put Landr Audio as the author, and add yourself in `contributors` property
https://docs.npmjs.com/files/package.json#people-fields-author-contributors

So:
```json
"author": {
        "name": "LANDR Audio Inc.",
        "email": "development@landr.com",
        "url": "https://github.com/LandrAudio"
    },
"contributors": [{
        "name": "Artem Diashkin",
        "email": "nemrosim1988@gmail.com",
        "url": "https://github.com/nemrosim"
    }],
```